starkbank/workspace/__workspace.py

```diff
@@ -72,22 +73,32 @@ def query(limit=None, username=None, ids=None, user=None):
     return rest.get_stream(resource=_resource, limit=limit, username=username, ids=ids, user=user)
 
 
-def update(id, username=None, name=None, allowed_tax_ids=None, user=None):
+def update(id, picture_type=None, username=None, name=None, allowed_tax_ids=None,  picture=None, user=None):
     """# Update Workspace entity
     Update a Workspace by passing its ID.
     ## Parameters (required):
     - id [string]: Workspace ID. ex: '5656565656565656'
+    ## Parameters (conditionally required):
+    - picture_type [string]: picture mime type. This parameter will be required if the picture parameter is informed ex: ‘image/png’ or ‘image/jpeg’
```

Comment:
- picture_type [string]: picture MIME type. This parameter will be required if the picture parameter is informed ex: "image/png" or "image/jpeg"